src/js/angular/core/directives/queryeditor/query-editor.directive.js

```diff
@@ -149,6 +149,10 @@ function queryEditorDirective($timeout, $location, toastr, $repositories, Sparql
             toastr.error('Cannot execute autocomplete query. ' + getError(data));
         };
 
+        window.editor.on("update", function () {
```

Comment:
Pff... it is not impossible to do it, but currently there is not view with more than one editor.
The implementation is like this (old code also) so the editor is accessable from the parent view controller.
I guess it could be reworked to emit the editor when it is ready so the controller can receive the editor.
It has to be checked in detail if it will work that way in all views.

Back to the question, if there are two editors in the view, only one will work as expected as other events are also used like this.